lib/friends/activity.rb

```diff
@@ -14,7 +14,11 @@ class Activity
 
     # @return [Regexp] the regex for capturing groups in deserialization
     def self.deserialization_regex
-      /(#{SERIALIZATION_PREFIX})?((?<date_s>\d{4}-\d\d-\d\d)(:\s)?)?(?<description>.+)?/
```

Comment:
This change is no good unfortunately; see https://github.com/JacobEvelyn/friends/issues/71. Fortunately, I just learned that you can selectively disable cops like so:

```ruby
def self.deserialization_regex
  # Note: This regex must be on one line because whitespace is important.
  # rubocop:disable Metrics/LineLength
  /really long regex ..... /
  # rubocop:enable Metrics/LineLength
end
```